code/game/objects/random/tools.dm

```diff
@@ -3,54 +3,55 @@
 	icon_state = "tool-grey"
 
 /obj/random/tool/item_to_spawn()
-	return pickweight(list(/obj/random/rare = 1,\
```

Comment:
I removed the trailing slashes from this file

but actual changes are just adding the advanced welder to this and the advanced list